IsInBedroom: 255
IsInBathroom: 46335
IsInInsideHallway: 2031360
IsInDiningArea: 16757760
IsInTVArea: 13434986
IsInKitchenArea: 60159